--- axbase-4.4.1.orig/usr/X11R6/bin/applixware
+++ axbase-4.4.1/usr/X11R6/bin/applixware
@@ -6,7 +6,7 @@
 #
 
 DBROOT=/dev/null
-APPLIX=/opt/applix/applix
+APPLIX=/usr/X11R6/lib/X11/applix/applix
 
 #
 # embedded jre is buggy
@@ -29,8 +29,8 @@
 #
 if [ ! -e ~/.odbc.ini ]; then
     echo "Copying .odbc.ini to $HOME"
-    test -f /usr/doc/packages/axbase/.odbc.ini && \
-	cp /usr/doc/packages/axbase/.odbc.ini ${HOME}/.odbc.ini
+    test -f /usr/share/doc/axbase/.odbc.ini && \
+	cp /usr/share/doc/axbase/.odbc.ini ${HOME}/.odbc.ini
 fi
 
 #
--- axbase-4.4.1.orig/debian/changelog
+++ axbase-4.4.1/debian/changelog
@@ -0,0 +1,10 @@
+axbase (4.4.1-14) unstable; urgency=low
+
+  * Converted from RPM binary format to debian.
+  * Made it not use /opt at all.
+
+ -- Joey Hess <joey@kitenet.net>  Wed,  7 Apr 1999 19:44:25 -0700
+
+Local variables:
+mode: debian-changelog
+End:
--- axbase-4.4.1.orig/debian/control
+++ axbase-4.4.1/debian/control
@@ -0,0 +1,12 @@
+Source: axbase
+Section: alien
+Priority: extra
+Maintainer: Joey Hess <joey@kitenet.net>
+
+Package: axbase
+Architecture: i386
+Depends: apxeng | apxgrm | apxfrn, axfont, ${shlibs:Depends}
+Description: base system of ApplixWare Office
+ This is the language independent part of ApplixWare Office for
+ linux. The language dependend part has to be installed in any
+ case. This package also contains Applix Builder and Applix Data.
--- axbase-4.4.1.orig/debian/copyright
+++ axbase-4.4.1/debian/copyright
@@ -0,0 +1,23 @@
+This package was debianized by the alien program by converting
+a binary RPM Package on Wed,  7 Apr 1999 19:44:25 -0700.
+
+Copyright: 1990-1998 Applix, Inc.
+
+Information from the binary Package:
+Name        : axbase                      Distribution: SuSE Linux 6.0-BETA (i386)
+Version     : 4.4.1                             Vendor: SuSE GmbH, Nuernberg, Germany
+Release     : 13                            Build Date: Tue Dec  1 01:36:41 1998
+Install date: (not installed)               Build Host: neumann.suse.de
+Group       : unsorted                      Source RPM: axbase-4.4.1-13.src.rpm
+Size        : 72974144                         License: 1990-1998 Applix, Inc.
+Packager    : feedback@suse.de
+Summary     : base system of ApplixWare Office
+Description :
+This is the language independent part of ApplixWare Office for
+linux. The language dependend part has to be installed in any
+case. This package also contains Applix Builder and Applix Data.
+
+Authors:
+--------
+    Eric Ding <ericding@applix.com>
+
--- axbase-4.4.1.orig/debian/rules
+++ axbase-4.4.1/debian/rules
@@ -0,0 +1,57 @@
+#!/usr/bin/make -f
+# debian/rules that uses debhelper and alien
+# GNU copyright 1997 by Joey Hess.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+build:
+	dh_testdir
+	# Nothing to do.
+
+clean:
+	dh_testdir
+	dh_testroot
+	dh_clean
+
+# Build architecture-independent files here.
+binary-indep: build
+# We have nothing to do by default.
+
+# Build architecture-dependent files here.
+binary-arch: build
+#	dh_testversion
+	dh_testdir
+	dh_testroot
+	dh_clean -k
+	dh_installdirs usr/X11R6/bin usr/X11R6/lib/X11
+	# Moving the files like this is a little nasty, but this way we
+	# need far less disk space to build the package..
+	mv opt/applix debian/tmp/usr/X11R6/lib/X11
+	cp -a usr/X11R6/bin/* debian/tmp/usr/X11R6/bin/
+	dh_installdocs usr/doc/packages/axbase/.*
+	dh_installexamples usr/doc/packages/axbase/contrib
+	dh_installmenu
+#	dh_installinit
+	dh_installcron
+#	dh_installmanpages
+#	dh_undocumented
+	dh_installchangelogs
+#	dh_strip
+	dh_compress
+	dh_fixperms
+	dh_suidregister
+	dh_installdeb
+	dh_shlibdeps
+	dh_gencontrol
+	dh_makeshlibs
+	dh_md5sums
+	dh_builddeb
+	# Undo the move.
+	mv debian/tmp/usr/X11R6/lib/X11/applix opt
+
+source diff:                                                                  
+	@echo >&2 'source and diff are obsolete - use dpkg-source -b'; false
+
+binary: binary-indep binary-arch
+.PHONY: build clean binary-indep binary-arch binary
--- axbase-4.4.1.orig/debian/README.Debian
+++ axbase-4.4.1/debian/README.Debian
@@ -0,0 +1,7 @@
+This is a converted RPM package.
+
+I decided to move the files out of /opt, since debian doesn't have an /opt
+yet. If the applix manual talks about files in /opt, look in
+/usr/X11R6/lib/X11/applix, instead.
+
+-- Joey Hess
--- axbase-4.4.1.orig/debian/menu
+++ axbase-4.4.1/debian/menu
@@ -0,0 +1 @@
+?package(axbase):needs="x11" section="Apps/Editors" title="Applixware" command="applixware"
