--- jdk-1.5.0.orig/debian/dirs
+++ jdk-1.5.0/debian/dirs
@@ -0,0 +1,4 @@
+usr/share/doc
+usr/share/man
+usr/lib/netscape/plugins-libc6
+usr/lib/mozilla/plugins
--- jdk-1.5.0.orig/debian/control
+++ jdk-1.5.0/debian/control
@@ -0,0 +1,24 @@
+Source: jdk
+Section: non-free/devel
+Priority: optional
+Maintainer: Gerald Turner <gturner@unzane.com>
+
+Package: jdk1.5
+Architecture: i386
+Depends: ${shlibs:Depends}
+Provides: java-common, java-virtual-machine, java-runtime, java2-runtime, java-browser-plugin, java-compiler, java2-compiler, j2sdk1.4, j2re1.4
+Suggests: mozilla-browser (>= 1.2.1-10)
+Description: Java(TM) 2 SDK, Standard Edition, Sun Microsystems(TM)
+ The Java(TM) 2 SDK is a development environment for building
+ applications, applets, and components that can be deployed on the
+ Java platform.
+ .
+ The Java(TM) 2 SDK software includes tools useful for developing and
+ testing programs written in the Java programming language and running
+ on the Java platform. These tools are designed to be used from the
+ command line. Except for appletviewer, these tools do not provide a
+ graphical user interface.
+ .
+ This package has been converted from a rpm package by alien version
+ 8.46 using a special diff file to help make the package conform to
+ debian policy written by Gerald Turner.
--- jdk-1.5.0.orig/debian/prerm
+++ jdk-1.5.0/debian/prerm
@@ -0,0 +1,35 @@
+#!/bin/sh
+
+case "`uname -m`" in
+    i[3-7]86 | ia32 | ia64)
+        ARCH=i386
+        ;;
+    sparc*)
+        ARCH=sparc
+        ;;
+    *)
+        ARCH="`uname -m`"
+        ;;
+esac
+
+if [ $1 = remove ]
+then
+    for i in appletviewer ControlPanel extcheck HtmlConverter idlj jar      \
+             jarsigner java javac javadoc javah javap java-rmi.cgi javaws   \
+             jconsole jdb jinfo jmap jps jsadebugd jstack jstat jstatd      \
+             keytool kinit klist ktab native2ascii orbd pack200 policytool  \
+             rmic rmid rmiregistry serialver servertool tnameserv unpack200
+    do
+        update-alternatives --remove $i /usr/lib/j2sdk-1.5.0/bin/$i
+    done
+
+    for i in apt
+    do
+        update-alternatives --remove j$i /usr/lib/j2sdk-1.5.0/bin/$i
+    done
+
+    update-alternatives \
+        --remove \
+        javaplugin_oji.so \
+        /usr/lib/j2sdk-1.5.0/jre/plugin/${ARCH}/ns7/libjavaplugin_oji.so
+fi
--- jdk-1.5.0.orig/debian/rules
+++ jdk-1.5.0/debian/rules
@@ -0,0 +1,108 @@
+#!/usr/bin/make -f
+# debian/rules for alien
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+# Use v4 compatability mode, so ldconfig gets added to maint scripts.
+export DH_COMPAT=4
+
+PACKAGE=$(shell dh_listpackages)
+
+JAVA_HOME := debian/$(PACKAGE)/usr/lib/j2sdk-1.5.0
+MAN_DIR := debian/$(PACKAGE)/usr/share/man
+
+build:
+	dh_testdir
+
+clean:
+	dh_testdir
+	dh_testroot
+	dh_clean
+
+binary-indep: build
+
+binary-arch: build
+	dh_testdir
+	dh_testroot
+	dh_clean -k
+	dh_installdirs
+
+# Copy the packages's files.
+	find . -maxdepth 1 -mindepth 1 -not -name debian -print0 | \
+		xargs -0 -r -i cp -a {} debian/$(PACKAGE)
+
+#
+# If you need to move files around in debian/$(PACKAGE) or do some
+# binary patching, do it here
+#
+
+# Move RPM contents to a proper debian/blackdown directory
+	mv debian/$(PACKAGE)/usr/java/jdk1.5.0 $(JAVA_HOME)
+	rmdir debian/$(PACKAGE)/usr/java
+
+# Move man pages to /usr/share/man
+	mv $(JAVA_HOME)/man/* $(MAN_DIR)
+	rmdir $(JAVA_HOME)/man
+
+# Move Japanese man pages further
+	rm $(MAN_DIR)/ja
+	mv $(MAN_DIR)/ja_JP.eucJP $(MAN_DIR)/ja
+	for i in `find $(MAN_DIR) -type f` ;                              \
+          do mv $$i `echo $$i | sed 's/\.\([^.]*\)$$/.j2sdk-1_5_0.\1/'` ; \
+        done
+
+# Unpack tripple compressed jars that showed up in 1.4.2
+	if [ -x $(JAVA_HOME)/bin/unpack200 ] ; then        \
+          for i in `find $(JAVA_HOME) -name \*.pack` ; do  \
+            $(JAVA_HOME)/bin/unpack200                     \
+              $$i `dirname $$i`/`basename $$i .pack`.jar ; \
+          done ;                                           \
+        fi
+
+# Fix ControlPanel shell script incompatibilities
+	if [ -f $(JAVA_HOME)/bin/ControlPanel ] ; then \
+          sed "s/\/usr\/bin\/ls/\/bin\/ls/g;           \
+	       s/expr \(.*\) : '^/expr \1 : '/;"       \
+            $(JAVA_HOME)/bin/ControlPanel              \
+	    > $(JAVA_HOME)/bin/ControlPanel.tmp ;      \
+          mv $(JAVA_HOME)/bin/ControlPanel.tmp         \
+             $(JAVA_HOME)/bin/ControlPanel ;           \
+          chmod 755 $(JAVA_HOME)/bin/ControlPanel ;    \
+        fi
+	if [ -f $(JAVA_HOME)/jre/bin/ControlPanel ] ; then \
+          sed "s/\/usr\/bin\/ls/\/bin\/ls/g;               \
+	       s/expr \(.*\) : '^/expr \1 : '/;"           \
+            $(JAVA_HOME)/jre/bin/ControlPanel              \
+	    > $(JAVA_HOME)/jre/bin/ControlPanel.tmp ;      \
+          mv $(JAVA_HOME)/jre/bin/ControlPanel.tmp         \
+             $(JAVA_HOME)/jre/bin/ControlPanel ;           \
+          chmod 755 $(JAVA_HOME)/jre/bin/ControlPanel ;    \
+        fi
+
+# Include optional documentation
+	if [ -f ../jdk-1_5_0-doc.zip ] ; then                            \
+            unzip -q -d debian/$(PACKAGE)/usr/share/doc                  \
+                  ../jdk-1_5_0-doc.zip ;                                 \
+            mv debian/$(PACKAGE)/usr/share/doc/docs                      \
+               debian/$(PACKAGE)/usr/share/doc/$(PACKAGE) ;              \
+        else                                                             \
+            echo jdk-1_5_0-doc.zip not found, skipping documentation   ; \
+        fi
+
+	dh_installdocs
+	dh_installchangelogs
+
+# This has been known to break on some wacky binaries.
+#	dh_strip
+	dh_compress
+	dh_fixperms
+	dh_makeshlibs
+	dh_installdeb
+	-dh_shlibdeps
+	dh_gencontrol
+	dh_md5sums
+	dh_builddeb
+
+binary: binary-indep binary-arch
+.PHONY: build clean binary-indep binary-arch binary
--- jdk-1.5.0.orig/debian/changelog
+++ jdk-1.5.0/debian/changelog
@@ -0,0 +1,92 @@
+jdk (1.5.0-1) unstable; urgency=low
+
+  * New upstream release
+
+ -- Gerald Turner <gturner@unzane.com>  Fri, 01 Oct 2004 14:19:16 -0700
+
+j2sdk (2:1.4.2_05-1) experimental; urgency=low
+
+  * New upstream release
+
+ -- Gerald Turner <gturner@unzane.com>  Fri, 01 Oct 2004 10:01:53 -0700
+
+j2sdk (2:1.4.2_03-1) experimental; urgency=low
+
+  * New upstream release
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Mon, 19 Jan 2004 12:33:32 -0800
+
+j2sdk (2:1.4.2-1) experimental; urgency=low
+
+  * Fix ControlPanel shell script incompatibilities.  Closes: #200731
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Thu, 10 Jul 2003 09:59:23 -0700
+
+j2sdk (1:1.4.2-1) experimental; urgency=low
+
+  * alien echoes the wrong filename, "j2sdk_1.4.2-1_i386.deb generated",
+    and fails to install when using the -i option.  Reverted package name
+    j2sdk1.4 to j2sdk.  Changed package version to 1 and use an epoch.
+    Closes: #199992
+  * The intent of the j2sdk1.4 package rename was so that that multiple
+    j2sdk versions could be installed simultaneously (i.e. j2sdk1.3).
+    This is no longer possible using this patch.
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Sun,  6 Jul 2003 13:39:31 -0700
+
+j2sdk (1.4.2-6) experimental; urgency=low
+
+  * New upstream release
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Mon, 30 Jun 2003 15:46:47 -0700
+
+j2sdk (1.4.1_02-5) experimental; urgency=low
+
+  * New upstream release
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Fri, 14 Mar 2003 14:43:45 -0800
+
+j2sdk (1.4.1_01-4) experimental; urgency=low
+
+  * New upstream release
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Tue, 14 Jan 2003 17:03:09 -0800
+
+j2sdk (1.4.1-3) experimental; urgency=low
+
+  * New upstream release
+  * Detection and installation of j2sdk-1_4_1-doc.zip (must be downloaded
+    and copied to the same directory where alien is executed)
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Wed,  2 Oct 2002 16:41:24 -0700
+
+j2sdk (1.4.1-2) experimental; urgency=low
+
+  * Added java-common Depends
+  * Added java-virtual-machine, java2-runtime, java-compiler, and
+    java2-compiler Provides
+  * Added netscape and mozilla Suggests
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Thu, 23 Aug 2002 09:07:09 -0700
+
+j2sdk (1.4.1-1) experimental; urgency=low
+
+  * New upstream release
+  * Detection and installation of j2sdk-1_4_1-rc-doc.zip (must be downloaded
+    and copied to the same directory where alien is executed)
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Thu, 22 Aug 2002 16:22:33 -0700
+
+j2sdk (1.4.0_01-1) experimental; urgency=low
+
+  * New upstream release
+  * Detection and installation of j2sdk-1_4_0-doc.zip (must be downloaded
+    and copied to the same directory where alien is executed)
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Thu, 22 Aug 2002 15:44:54 -0700
+
+j2sdk (1.4.0-1) experimental; urgency=low
+
+  * Converted from .rpm format to .deb
+
+ -- Gerald Turner <gturner@newedgenetworks.com>  Wed, 22 May 2002 12:46:34 -0700
--- jdk-1.5.0.orig/debian/postinst
+++ jdk-1.5.0/debian/postinst
@@ -0,0 +1,111 @@
+#!/bin/sh
+
+case "`uname -m`" in
+    i[3-7]86 | ia32 | ia64)
+        ARCH=i386
+        ;;
+    sparc*)
+        ARCH=sparc
+        ;;
+    *)
+        ARCH="`uname -m`"
+        ;;
+esac
+
+if [ $1 = configure ]
+then
+    for i in appletviewer ControlPanel extcheck HtmlConverter idlj jar      \
+             jarsigner java javac javadoc javah javap java-rmi.cgi javaws   \
+             jconsole jdb jinfo jmap jps jsadebugd jstack jstat jstatd      \
+             keytool kinit klist ktab native2ascii orbd pack200 policytool  \
+             rmic rmid rmiregistry serialver servertool tnameserv unpack200
+    do
+        if [ -e /usr/share/man/man1/$i.j2sdk-1_5_0.1.gz ]
+        then
+            if [ -e /usr/share/man/ja/man1/$i.j2sdk-1_5_0.1.gz ]
+            then
+                update-alternatives \
+                    --install \
+                        /usr/bin/$i \
+                        $i \
+                        /usr/lib/j2sdk-1.5.0/bin/$i \
+                        1500 \
+                    --slave \
+                        /usr/share/man/man1/$i.1.gz \
+                        $i.1.gz \
+                        /usr/share/man/man1/$i.j2sdk-1_5_0.1.gz \
+                    --slave \
+                        /usr/share/man/ja/man1/$i.1.gz \
+                        $i.ja.1.gz \
+                        /usr/share/man/ja/man1/$i.j2sdk-1_5_0.1.gz
+            else
+                update-alternatives \
+                    --install \
+                        /usr/bin/$i \
+                        $i \
+                        /usr/lib/j2sdk-1.5.0/bin/$i \
+                        1500 \
+                    --slave \
+                        /usr/share/man/man1/$i.1.gz \
+                        $i.1.gz \
+                        /usr/share/man/man1/$i.j2sdk-1_5_0.1.gz
+            fi
+        else
+            update-alternatives \
+                --install \
+                    /usr/bin/$i \
+                    $i \
+                    /usr/lib/j2sdk-1.5.0/bin/$i \
+                    1500
+        fi
+    done
+
+    for i in apt
+    do
+        if [ -e /usr/share/man/man1/$i.j2sdk-1_5_0.1.gz ]
+        then
+            if [ -e /usr/share/man/ja/man1/$i.j2sdk-1_5_0.1.gz ]
+            then
+                update-alternatives \
+                    --install \
+                        /usr/bin/j$i \
+                        j$i \
+                        /usr/lib/j2sdk-1.5.0/bin/$i \
+                        1500 \
+                    --slave \
+                        /usr/share/man/man1/j$i.1.gz \
+                        j$i.1.gz \
+                        /usr/share/man/man1/$i.j2sdk-1_5_0.1.gz \
+                    --slave \
+                        /usr/share/man/ja/man1/j$i.1.gz \
+                        j$i.ja.1.gz \
+                        /usr/share/man/ja/man1/$i.j2sdk-1_5_0.1.gz
+            else
+                update-alternatives \
+                    --install \
+                        /usr/bin/j$i \
+                        j$i \
+                        /usr/lib/j2sdk-1.5.0/bin/$i \
+                        1500 \
+                    --slave \
+                        /usr/share/man/man1/j$i.1.gz \
+                        j$i.1.gz \
+                        /usr/share/man/man1/$i.j2sdk-1_5_0.1.gz
+            fi
+        else
+            update-alternatives \
+                --install \
+                    /usr/bin/j$i \
+                    j$i \
+                    /usr/lib/j2sdk-1.5.0/bin/$i \
+                    1500
+        fi
+    done
+
+    update-alternatives \
+        --install \
+        /usr/lib/mozilla/plugins/javaplugin_oji.so \
+        javaplugin_oji.so \
+        /usr/lib/j2sdk-1.5.0/jre/plugin/${ARCH}/ns7/libjavaplugin_oji.so \
+        1500
+fi
--- jdk-1.5.0.orig/debian/copyright
+++ jdk-1.5.0/debian/copyright
@@ -0,0 +1,25 @@
+This package was debianized by the alien program by converting
+a binary .rpm package on Fri,  1 Oct 2004 10:18:23 -0700
+
+Copyright: Sun Microsystems Binary Code License (BCL)
+
+Information from the binary package:
+Name        : jdk                          Relocations: /usr/java 
+Version     : 1.5.0                             Vendor: Sun Microsystems, Inc.
+Release     : fcs                           Build Date: Wed 15 Sep 2004 06:18:37 AM PDT
+Install date: (not installed)               Build Host: tiger-linux
+Group       : Development/Tools             Source RPM: jdk-1.5.0-fcs.src.rpm
+Size        : 80132114                         License: Sun Microsystems Binary Code License (BCL)
+Packager    : Java Software <j2se-comments@java.sun.com>
+URL         : http://java.sun.com/
+Summary     : Java(TM) 2 Platform Standard Edition Development Kit
+Description :
+The Java 2 Platform Standard Edition Development Kit (JDK) includes both the
+runtime environment (Java virtual machine, the Java platform classes and
+supporting files) and development tools (compilers, debuggers, tool libraries
+and other tools).
+
+The JDK is a development environment for building applications, applets and
+components that can be deployed with the Java 2 Platform Standard Edition
+Runtime Environment.
+
