--- motif-2.1.10.orig/debian/control
+++ motif-2.1.10/debian/control
@@ -0,0 +1,18 @@
+Source: motif
+Section: non-free/x11
+Priority: extra
+Build-Depends: debhelper
+Maintainer: Andreas Voegele <andreas.voegele@gmx.de>
+Standards-Version: 3.1.1
+
+Package: motif
+Architecture: any
+Depends: libc6, xlibs
+Conflicts: lesstif-bin, lesstif-dev, lesstif-dbg, lesstif1, lesstifg, openmotif, openmotif-devel, suidmanager (<< 0.50)
+Provides: x-window-manager
+Suggests: menu
+Description: Runtime libraries and executables for the Motif toolkit
+ This package contains:
+   - The Motif shared libraries, needed to run applications which are
+     `dynamically linked' with Motif.
+   - The popular Motif Window Manager, `mwm'.
--- motif-2.1.10.orig/debian/copyright
+++ motif-2.1.10/debian/copyright
@@ -0,0 +1,150 @@
+This is the prepackaged version of Metro Link Motif. It was debianized
+by the Alien program by converting a binary RPM package.
+
+Metro Link's web site can be found at http://www.metrolink.com/
+
+METRO LINK WARRANTY AND LICENSE AGREEMENT
+ 
+READ THESE LICENSE TERMS BEFORE OPENING THE SOFTWARE PACKAGE.  
+OPENING THE SOFTWARE PACKAGE INDICATES YOUR ACCEPTANCE OF THIS LICENSE.  
+IF YOU DO NOT AGREE TO THESE LICENSE TERMS, RETURN THE SOFTWARE PACKAGE
+UNOPENED (ALONG WITH ALL ASSOCIATED WRITTEN PRODUCT MATERIALS) AND 
+WITHIN THE WARRANTY PERIOD TO THE DEALER FROM WHOM YOU OBTAINED IT 
+FOR A FULL REFUND.
+
+This License is an agreement between you and METRO LINK INCORPORATED 
+(hereafter "ML").
+
+1.  LICENSE GRANT
+ 
+Subject to the terms set forth in this License, you may use the 
+enclosed computer product ("SOFTWARE"), including all associated 
+software documentation furnished with this package, solely for your 
+internal business purposes in the country in which the Software was 
+furnished to you and on a single computer system (i.e. single or 
+multiple CPU system in the same physical cabinet).
+
+NFS mounting of programs or clients is not permitted.
+ 
+
+2.  COPYRIGHT AND TITLE.
+ 
+The Software and its copyrights are owned by ML or its suppliers.  
+The Software is protected by United States copyright laws and 
+international treaty provisions.  Therefore, you must treat the 
+Software like any other copyrighted material (e.g., a book or musical
+recording) except that you may either (a) make one (1) copy of the 
+Software solely for backup purposes or (b) transfer the Software to 
+a single hard disk provided you keep the original solely for backup 
+or archival purposes.  You must reproduce and include the copyright 
+notice on the backup copy.  You may not copy the written materials.
+ 
+
+3.  RESTRICTED USE.
+ 
+You may not rent or lease the Software, and you may not reverse engineer, 
+decompile, or disassemble the Software.  Furthermore, certain components
+of the Software are licensed for use exclusively with hardware supplied
+by specific manufacturers, and you may not modify the Software so that
+it will operate with hardware supplied by alternate manufacturers.
+ 
+
+4.  ADAPTATIONS
+ 
+Any part of the Software that you merge with other software is still 
+property of ML.  In the event this License is terminated, you may not 
+use part of the Software even if it has been merged with other software.  
+You agree to take reasonable steps to protect ML's Software from theft 
+or use contrary to this License.
+ 
+
+5.  LIMITED 30-DAY WARRANTIES.
+ 
+ML warrants to you (the original Licensee) for thirty (30) days from 
+the date of delivery to you of the Software, as evidenced by a copy 
+of your license purchase receipt ( the Warranty Period"), that the 
+Software shall be in operable condition as described in the documentation
+provided with the product code, and that the media upon which the 
+Software is furnished to you will be free from defects in material and
+workmanship under normal use.  FOR WARRANTY ASSISTANCE, YOU MAY EITHER 
+RETURN THE SOFTWARE TO THE DEALER FROM WHICH YOU PURCHASED THE LICENSE, 
+OR CONTACT THAT DEALER (WHOSE NAME AND ADDRESS IS SET FORTH ON THE BOX
+CONTAINING THIS WARRANTY), OR CONTACT ML AT THE ADDRESS STATED ON THIS 
+LICENSE, ATTN: CUSTOMER SERVICE DEPARTMENT.  This warranty allocates
+risks of product failure between you and ML.
+ 
+
+6.  NO OTHER WARRANTIES.
+ 
+The agents, employees, distributors, and dealers of ML are not 
+authorized to make modifications to this warranty, nor are additional
+warranties binding on ML.  Accordingly, additional statements such as 
+dealer advertising or presentations, whether oral or written, do not 
+constitute warranties by ML and should not be relied upon as a limited 
+warranty of ML.
+
+ML AND ITS SUPPLIERS MAKE NO OTHER REPRESENTATIONS, CONDITIONS, OR 
+WARRANTIES, EXPRESSED OR IMPLIED.  ANY IMPLIED WARRANTY OR CONDITION OF
+MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE IS LIMITED IN 
+DURATION TO THE WARRANTY PERIOD.  SOME STATES DO NOT ALLOW LIMITATIONS 
+ON HOW LONG AN IMPLIED WARRANTY LASTS, SO THE ABOVE LIMITATIONS MAY NOT 
+APPLY TO YOU.  THIS WARRANTY GIVES YOU SPECIFIC LEGAL RIGHTS AND YOU MAY 
+ALSO HAVE OTHER LEGAL RIGHTS WHICH VARY FROM STATE TO STATE.
+ 
+
+7.  CUSTOMER REMEDIES
+ 
+YOUR SOLE AND EXCLUSIVE REMEDY AND ML'S SOLE LIABILITY HEREUNDER 
+SHALL BE, UPON YOUR RETURN OF THE SOFTWARE TO ML WITH THE RECEIPT 
+WITHIN THE WARRANTY PERIOD, AT ML'S OPTION, REPLACEMENT OR REPAIR 
+OF SOFTWARE THAT DOES NOT MEET ML'S LIMITED WARRANTY OR A REFUND OF 
+THE SOFTWARE'S PURCHASE PRICE.  ML'S SUPPLIERS SHALL NOT BE HELD TO 
+ANY LIABILITY FOR ANY DAMAGES SUFFERED OR INCURRED BY YOU.  IN NO EVENT 
+SHALL ML OR ITS SUPPLIERS BE LIABLE TO YOU OR ANY OTHER PERSON FOR ANY 
+INCIDENTAL, COLLATERAL, SPECIAL, OR CONSEQUENTIAL DAMAGES, INCLUDING 
+WITHOUT LIMITATION, DAMAGES FOR LOSS OF PROFITS, LOSS OF CUSTOMERS, 
+LOSS OF GOOD WILL, WORK STOPPAGE, DATA LOSS, COMPUTER FAILURE OR 
+MALFUNCTION, CLAIMS BY ANY PARTY OTHER THAN YOU, OR ANY OR ALL OTHER 
+SIMILAR DAMAGES OR LOSS EVEN IF ML, ITS SUPPLIERS, OR ITS AGENT HAS 
+BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.  SOME STATES DO NOT 
+ALLOW THE EXCLUSION OR LIMITATION OF INCIDENTAL OR CONSEQUENTIAL 
+DAMAGES, SO THE ABOVE LIMITATIONS OR EXCLUSION MAY NOT APPLY TO YOU.
+
+ML shall have no liability or responsibility for Software altered, 
+modified, or converted by you or a third party, damages resulting 
+from accident, abuse, or misapplication, or for problems due to the 
+malfunction of your equipment or software not supplied by ML.
+ 
+
+8.  TERMINATION
+ 
+This License is in effect until terminated and terminates without 
+notice from ML if you fail to comply with any of its provisions.  
+Upon termination you shall either destroy the Software and all copies 
+or portions thereof or return the Software and all copies as portions 
+thereof to ML.
+ 
+
+9.  GENERAL
+ 
+This License is governed by the laws of the State of Florida.  This 
+License represents the entire statement of the understandings and 
+agreements among the parties.
+ 
+
+10. U.S. GOVERNMENT RESTRICTED RIGHTS
+ 
+This Software and documentation is provided with RESTRICTED RIGHTS.  
+Use, duplication, or disclosure by the Government is subject to 
+restrictions as set forth in subparagraph (c)(1)(ii) of the Rights 
+in Technical Data and Computer FAR 52.227-14 (Alternate III) or the 
+Software clause at DFAR 252.227-7013.  Contractor/manufacturer is 
+METRO LINK INCORPORATED, 4711 North Powerline Road, Fort Lauderdale, 
+FL  33309.
+
+Metro Link Motif Complete!, Metro Link Motif, Metro-X, Metro
+Extreme 3D, Metro OpenGL, Metro-Xv, and Metro-XIE are trademarks
+of Metro Link Incorporated.
+
+XRT, XRT/graph, XRT/field, XRT/gear, XRT/3d, and XRT/table are
+trademarks of KL Group Incorporated.
--- motif-2.1.10.orig/debian/rules
+++ motif-2.1.10/debian/rules
@@ -0,0 +1,113 @@
+#!/usr/bin/make -f
+# debian/rules that uses debhelper and alien
+# GNU copyright 1997 by Joey Hess.
+
+SHELL = /bin/sh
+
+package := $(shell basename `pwd` | sed 's/-[0-9].*//')
+version := $(shell basename `pwd` | tr -cd '[0-9.]')
+major := $(word 1,$(subst ., ,$(version)))
+minor := $(word 2,$(subst ., ,$(version)))
+
+mwmrc := etc/X11/mwm/system.mwmrc
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+# This is the debhelper compatability version to use.
+export DH_COMPAT=1
+
+motif-version:
+	sed -e 's/#PACKAGE#/$(package)/' -e 's/#VERS#/$(version)/' debian/changelog-template >debian/changelog
+
+build: motif-version
+	dh_testdir
+
+clean: motif-version
+	dh_testdir
+	dh_testroot
+	dh_clean debian/conffiles debian/menu
+
+# Build architecture-independent files here.
+binary-indep: build
+# We have nothing to do by default.
+
+# Build architecture-dependent files here.
+binary-arch: build
+#	dh_testversion
+	dh_testdir
+	dh_testroot
+	dh_clean -k
+	dh_installdirs
+	tar -cf - --exclude=debian --exclude='./usr/doc' --exclude='*.so.?' --exclude='*.so' . | tar -xpf - -C debian/tmp
+#
+# If you need to move files around in debian/tmp or do some
+# binary patching ... Insert it here
+#
+	-ln -fs libMrm.so.$(major).$(minor) debian/tmp/usr/X11R6/lib/libMrm.so.$(major)
+	-ln -fs libUil.so.$(major).$(minor) debian/tmp/usr/X11R6/lib/libUil.so.$(major)
+	-ln -fs libXm.so.$(major).$(minor) debian/tmp/usr/X11R6/lib/libXm.so.$(major)
+# Move system.mwmrc to /etc/X11/mwm/system.mwmrc.
+	@mwmrc=`find debian/tmp -name system.mwmrc -type f -printf '%P\t' | cut -f1`; \
+	if [ "$$mwmrc" != $(mwmrc) ]; then \
+	  echo "Moving /$$mwmrc to $(mwmrc)"; \
+	  mv debian/tmp/$$mwmrc debian/tmp/$(mwmrc); \
+	  dh_link $(mwmrc) $$mwmrc; \
+	fi
+# Move /usr/X11R6/lib/X11/app-defaults to /etc/X11/app-defaults.
+	@if [ -d debian/tmp/usr/X11R6/lib/X11/app-defaults ]; then \
+	  echo "Moving /usr/X11R6/lib/X11/app-defaults to /etc/X11/app-defaults"; \
+	  mv debian/tmp/usr/X11R6/lib/X11/app-defaults debian/tmp/etc/X11; \
+	fi
+# Install the menu method and /etc/X11/mwm/system.mwmrc-menu.
+	@echo "Installing menu method"
+	@sed -e 's/#PACKAGE#/$(package)/' debian/menu-template >debian/menu
+	@install -m 0755 debian/mwm-menumethod debian/tmp/etc/menu-methods
+	@cp debian/etcmwmrc debian/tmp/$(mwmrc)-menu
+	@echo "include-menu-defs" >>debian/tmp/$(mwmrc)-menu
+# Find the configuration files.
+	@echo "Looking for configuration files"
+	@echo "/$(mwmrc)-menu" >debian/conffiles
+	@find debian/tmp/etc -name system.fpanelrc -type f -printf '/etc/%P\n' >>debian/conffiles
+	@if [ -d debian/tmp/etc/X11/app-defaults ]; then \
+	  find debian/tmp/etc/X11/app-defaults -type f -printf '/etc/X11/app-defaults/%P\n' >>debian/conffiles; \
+	fi
+# Fix mxterm.
+	@if [ -f debian/tmp/usr/X11R6/bin/mxterm ]; then \
+	  echo "Fixing mxterm"; \
+	  perl -pi -e 's/libtermcap\.so\.2/libncurses.so.5/' debian/tmp/usr/X11R6/bin/mxterm; \
+	fi
+# Copy documentation.
+	@if [ -d usr/doc ]; then \
+	  echo "Copying documentation"; \
+	  mkdir -p debian/tmp/usr/share/doc/$(package); \
+	  cp -a usr/doc/*/* debian/tmp/usr/share/doc/$(package); \
+	fi
+	dh_installwm --priority=40 mwm
+	dh_installdocs
+	dh_installexamples
+	dh_installmenu
+#	dh_installinit
+#	dh_installcron
+#	dh_installmanpages
+#	dh_undocumented
+	dh_installchangelogs
+	dh_strip
+	dh_compress
+	dh_fixperms
+	@if [ -f debian/tmp/usr/X11R6/bin/mxterm ]; then \
+	  chown root:utmp debian/tmp/usr/X11R6/bin/mxterm; \
+	  chmod 2755 debian/tmp/usr/X11R6/bin/mxterm; \
+	fi
+	dh_installdeb
+#	-dh_shlibdeps
+	dh_gencontrol
+	dh_makeshlibs
+	dh_md5sums
+	dh_builddeb
+
+source diff:                                                                  
+	@echo >&2 'source and diff are obsolete - use dpkg-source -b'; false
+
+binary: binary-indep binary-arch
+.PHONY: build clean binary-indep binary-arch binary motif-version
--- motif-2.1.10.orig/debian/mwm-menumethod
+++ motif-2.1.10/debian/mwm-menumethod
@@ -0,0 +1,30 @@
+#!/usr/sbin/install-menu
+#   -*- mode: shell-script; -*-
+#I need menu-1!
+#
+#NOTE: the first line of this script _must_ be
+# equal to "#!/usr/sbin/install-menu", otherwise update-menus
+# will feed this script old-compat-mode data.
+#
+#More info: /usr/share/doc/menu/README.
+#
+compat="menu-1"
+
+!include menu.h
+
+function mwmcommon($c)=\
+   "\t\"" title() "\"\tf.exec  \"" replacewith($c,"\"","'") " &\"\n"
+
+supported 
+  x11 =mwmcommon($command)
+  text=mwmcommon(term())
+endsupported
+startmenu=   "Menu \"" $section "\"\n{\n\t\"" \
+              ifelse(title(),title(),"Debian") "\"\tf.title\n"
+endmenu=     "}\n"
+submenutitle="  \"" title() "\" f.menu \"" $section "\"\n"
+genmenu=      "menudefs.hook"
+rcfile=       "system.mwmrc"
+examplercfile="system.mwmrc-menu"
+rootprefix=   "/etc/X11/mwm/"
+userprefix=   "/.mwm/"
--- motif-2.1.10.orig/debian/etcmwmrc
+++ motif-2.1.10/debian/etcmwmrc
@@ -0,0 +1,103 @@
+!!
+!! Root Menu Description
+!!
+
+Menu DefaultRootMenu
+{
+	"Root Menu"		f.title
+	"Debian"                f.menu "/Debian"
+	"New Window"		f.exec "x-terminal-emulator &"
+	"Shuffle Up"		f.circle_up
+	"Shuffle Down"		f.circle_down
+	"Refresh"		f.refresh
+	"Pack Icons"		f.pack_icons
+!	"Toggle Behavior..."	f.set_behavior
+	 no-label		f.separator
+	"Restart..."		f.restart
+	"Quit..."		f.quit_mwm
+}
+
+!!
+!! Default Window Menu
+!!
+Menu DefaultWindowMenu
+{
+	"Make Pinned"				f.toggle_pin_state
+	no-label				f.separator
+	Restore		_R	Alt<Key>F5	f.restore
+	Move		_M	Alt<Key>F7	f.move
+	Size		_S	Alt<Key>F8	f.resize
+	Minimize	_n	Alt<Key>F9	f.minimize
+	Maximize	_x	Alt<Key>F10	f.maximize
+	Raise		_a	Alt<Key>F2	f.raise
+	Lower		_L	Alt<Key>F3	f.lower
+	no-label				f.separator
+	Close		_C	Alt<Key>F4	f.kill
+}
+
+
+!!
+!! Default Key Bindings
+!!
+Keys DefaultKeyBindings
+{
+	Shift<Key>Escape	window|icon		f.post_wmenu
+	Alt<Key>space		window|icon		f.post_wmenu
+	Alt<Key>Tab		root|icon|window	f.next_key
+	Alt Shift<Key>Tab	root|icon|window	f.prev_key
+	Alt<Key>Escape		root|icon|window	f.circle_down
+	Alt Shift<Key>Escape	root|icon|window	f.circle_up
+	Alt Shift Ctrl<Key>exclam root|icon|window	f.set_behavior
+	Alt<Key>F6		window			f.next_key transient
+	Alt Shift<Key>F6	window			f.prev_key transient
+	Shift<Key>F10		icon			f.post_wmenu
+	Alt Shift<Key>Delete	root|icon|window	f.restart
+}
+
+!!
+!! Button Binding Description(s)
+!!
+!! These can be used anywhere
+!!
+Buttons DefaultButtonBindings
+{
+	<Btn1Down>	icon|frame	f.raise
+	<Btn3Down>	icon|frame	f.post_wmenu
+	<Btn3Down>	root		f.menu	DefaultRootMenu
+}
+ 
+!!
+!! These are some example bindings that you might use if your
+!! keyboardFocusPolicy is "explicit".  Note that to use these, you'll
+!! have to change the Mwm*buttonBindings resource in Mwm to
+!! ExplicitButtonBindings instead of DefaultButtonBindings.
+!!
+Buttons ExplicitButtonBindings
+{
+	<Btn1Down>	frame|icon	f.raise
+	<Btn3Down>	frame|icon	f.post_wmenu
+	<Btn3Down>	root		f.menu	DefaultRootMenu
+!	<Btn1Up>	icon		f.restore
+	Alt<Btn1Down>	window|icon	f.lower
+!	Alt<Btn2Down>	window|icon	f.resize
+!	Alt<Btn3Down>	window|icon	f.move
+}
+
+!! These are some example bindings that you might use if your
+!! keyboardFocusPolicy is "pointer".  Note that to use these, you'll
+!! have to change the Mwm*buttonBindings resource in Mwm to
+!! PointerButtonBindings instead of DefaultButtonBindings.
+!!
+Buttons PointerButtonBindings
+{
+	<Btn1Down>	frame|icon	f.raise
+	<Btn3Down>	frame|icon	f.post_wmenu
+	<Btn3Down>	root		f.menu	DefaultRootMenu
+	<Btn1Down>	window		f.raise
+!	<Btn1Up>	icon		f.restore
+	Alt<Btn1Down>	window|icon	f.lower
+!	Alt<Btn2Down>	window|icon	f.resize
+!	Alt<Btn3Down>	window|icon	f.move
+}
+
+!include-menu-defs
--- motif-2.1.10.orig/debian/dirs
+++ motif-2.1.10/debian/dirs
@@ -0,0 +1,4 @@
+etc/X11/mwm
+etc/menu-methods
+usr/X11R6/bin
+usr/X11R6/lib
--- motif-2.1.10.orig/debian/changelog-template
+++ motif-2.1.10/debian/changelog-template
@@ -0,0 +1,37 @@
+#PACKAGE# (#VERS#-7) unstable; urgency=low
+
+  * Mwm is registered with dh_installwm.
+  * Disabled dh_undocumented.
+  * Removed dh_suidregister.
+  * The virtual package `x-terminal-emulator' is no longer provided since
+    Metro Link's latest Motif release doesn't contain mxterm anymore.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Tue, 16 Jan 2001 12:17:38 +0100
+
+#PACKAGE# (#VERS#-6) unstable; urgency=low
+
+  * Moved app-defaults directory to /etc/X11.
+  * Depends on xlibs.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Sat,  9 Dec 2000 08:45:11 +0100
+
+#PACKAGE# (#VERS#-5) unstable; urgency=low
+
+  * Made mxterm use libncurses.so.5 instead of libtermcap.so.2.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Mon,  3 Jan 2000 09:50:03 +0100
+
+#PACKAGE# (#VERS#-4) unstable; urgency=low
+
+  * Converted from RPM binary format to Debian.
+  * Removed *.so symlinks.
+  * Made mxterm use libncurses.so.4 instead of libtermcap.so.2.
+  * Enabled dh_undocumented, dh_fixperms and dh_strip.
+  * Added menu file and menu method.
+  * Added maintainer scripts.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Tue, 16 Nov 1999 19:31:03 +0100
+
+Local variables:
+mode: debian-changelog
+End:
--- motif-2.1.10.orig/debian/postinst
+++ motif-2.1.10/debian/postinst
@@ -0,0 +1,7 @@
+#!/bin/sh -e
+
+if [ "$1" = "configure" ]; then
+  ldconfig
+fi
+
+#DEBHELPER#
--- motif-2.1.10.orig/debian/README.Debian
+++ motif-2.1.10/debian/README.Debian
@@ -0,0 +1,10 @@
+Motif for Debian
+----------------
+
+For compatibility with Lesstif the Motif window manager configuration
+file system.mwmrc is placed in /etc/X11/mwm on Debian systems.
+
+If the `menu' package is installed, system.mwmrc is automatically
+generated from /etc/X11/mwm/system.mwmrc-menu.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Tue, 16 Nov 1999 19:31:03 +0100
--- motif-2.1.10.orig/debian/menu-template
+++ motif-2.1.10/debian/menu-template
@@ -0,0 +1,6 @@
+?package(#PACKAGE#):\
+ needs="wm"\
+ section="WindowManagers"\
+ title="Mwm"\
+ longtitle="Motif window manager"\
+ command="/usr/X11R6/bin/mwm"
