--- motif-devel-2.1.10.orig/debian/changelog-template
+++ motif-devel-2.1.10/debian/changelog-template
@@ -0,0 +1,31 @@
+#PACKAGE# (#VERS#-7) unstable; urgency=low
+
+  * Disabled dh_undocumented.
+  * Removed dh_suidregister.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Tue, 16 Jan 2001 11:58:56 +0100
+
+#PACKAGE# (#VERS#-6) unstable; urgency=low
+
+  * Depends on xlibs-dev.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Sat,  9 Dec 2000 12:27:50 +0100
+
+#PACKAGE# (#VERS#-5) unstable; urgency=low
+
+  * Made mxterm use libncurses.so.5 instead of libtermcap.so.2.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Mon,  3 Jan 2000 10:04:26 +0100
+
+#PACKAGE# (#VERS#-4) unstable; urgency=low
+
+  * Converted from RPM binary format to Debian.
+  * Added *.so symlinks.
+  * Added mxmkmf.
+  * Enabled dh_undocumented and dh_fixperms.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Tue, 16 Nov 1999 19:31:18 +0100
+
+Local variables:
+mode: debian-changelog
+End:
--- motif-devel-2.1.10.orig/debian/control
+++ motif-devel-2.1.10/debian/control
@@ -0,0 +1,14 @@
+Source: motif-devel
+Section: non-free/x11
+Priority: extra
+Build-Depends: debhelper, motif
+Maintainer: Andreas Voegele <andreas.voegele@gmx.de>
+Standards-Version: 3.1.1
+
+Package: motif-devel
+Architecture: any
+Depends: motif (= ${Source-Version}), libc6-dev, xlibs-dev
+Conflicts: lesstif-bin, lesstif-dev, lesstif-dbg, lesstif1, lesstifg, openmotif, openmotif-devel
+Description: Development files for the Motif toolkit
+ This is the Motif development environment.  It includes the static
+ libraries and header files necessary to build Motif based applications.
--- motif-devel-2.1.10.orig/debian/copyright
+++ motif-devel-2.1.10/debian/copyright
@@ -0,0 +1,150 @@
+This is the prepackaged version of Metro Link Motif. It was debianized
+by the alien program by converting a binary RPM package.
+
+Metro Link's web site can be found at http://www.metrolink.com/
+
+METRO LINK WARRANTY AND LICENSE AGREEMENT
+ 
+READ THESE LICENSE TERMS BEFORE OPENING THE SOFTWARE PACKAGE.  
+OPENING THE SOFTWARE PACKAGE INDICATES YOUR ACCEPTANCE OF THIS LICENSE.  
+IF YOU DO NOT AGREE TO THESE LICENSE TERMS, RETURN THE SOFTWARE PACKAGE
+UNOPENED (ALONG WITH ALL ASSOCIATED WRITTEN PRODUCT MATERIALS) AND 
+WITHIN THE WARRANTY PERIOD TO THE DEALER FROM WHOM YOU OBTAINED IT 
+FOR A FULL REFUND.
+
+This License is an agreement between you and METRO LINK INCORPORATED 
+(hereafter "ML").
+
+1.  LICENSE GRANT
+ 
+Subject to the terms set forth in this License, you may use the 
+enclosed computer product ("SOFTWARE"), including all associated 
+software documentation furnished with this package, solely for your 
+internal business purposes in the country in which the Software was 
+furnished to you and on a single computer system (i.e. single or 
+multiple CPU system in the same physical cabinet).
+
+NFS mounting of programs or clients is not permitted.
+ 
+
+2.  COPYRIGHT AND TITLE.
+ 
+The Software and its copyrights are owned by ML or its suppliers.  
+The Software is protected by United States copyright laws and 
+international treaty provisions.  Therefore, you must treat the 
+Software like any other copyrighted material (e.g., a book or musical
+recording) except that you may either (a) make one (1) copy of the 
+Software solely for backup purposes or (b) transfer the Software to 
+a single hard disk provided you keep the original solely for backup 
+or archival purposes.  You must reproduce and include the copyright 
+notice on the backup copy.  You may not copy the written materials.
+ 
+
+3.  RESTRICTED USE.
+ 
+You may not rent or lease the Software, and you may not reverse engineer, 
+decompile, or disassemble the Software.  Furthermore, certain components
+of the Software are licensed for use exclusively with hardware supplied
+by specific manufacturers, and you may not modify the Software so that
+it will operate with hardware supplied by alternate manufacturers.
+ 
+
+4.  ADAPTATIONS
+ 
+Any part of the Software that you merge with other software is still 
+property of ML.  In the event this License is terminated, you may not 
+use part of the Software even if it has been merged with other software.  
+You agree to take reasonable steps to protect ML's Software from theft 
+or use contrary to this License.
+ 
+
+5.  LIMITED 30-DAY WARRANTIES.
+ 
+ML warrants to you (the original Licensee) for thirty (30) days from 
+the date of delivery to you of the Software, as evidenced by a copy 
+of your license purchase receipt ( the Warranty Period"), that the 
+Software shall be in operable condition as described in the documentation
+provided with the product code, and that the media upon which the 
+Software is furnished to you will be free from defects in material and
+workmanship under normal use.  FOR WARRANTY ASSISTANCE, YOU MAY EITHER 
+RETURN THE SOFTWARE TO THE DEALER FROM WHICH YOU PURCHASED THE LICENSE, 
+OR CONTACT THAT DEALER (WHOSE NAME AND ADDRESS IS SET FORTH ON THE BOX
+CONTAINING THIS WARRANTY), OR CONTACT ML AT THE ADDRESS STATED ON THIS 
+LICENSE, ATTN: CUSTOMER SERVICE DEPARTMENT.  This warranty allocates
+risks of product failure between you and ML.
+ 
+
+6.  NO OTHER WARRANTIES.
+ 
+The agents, employees, distributors, and dealers of ML are not 
+authorized to make modifications to this warranty, nor are additional
+warranties binding on ML.  Accordingly, additional statements such as 
+dealer advertising or presentations, whether oral or written, do not 
+constitute warranties by ML and should not be relied upon as a limited 
+warranty of ML.
+
+ML AND ITS SUPPLIERS MAKE NO OTHER REPRESENTATIONS, CONDITIONS, OR 
+WARRANTIES, EXPRESSED OR IMPLIED.  ANY IMPLIED WARRANTY OR CONDITION OF
+MERCHANTABILITY OR FITNESS FOR A PARTICULAR PURPOSE IS LIMITED IN 
+DURATION TO THE WARRANTY PERIOD.  SOME STATES DO NOT ALLOW LIMITATIONS 
+ON HOW LONG AN IMPLIED WARRANTY LASTS, SO THE ABOVE LIMITATIONS MAY NOT 
+APPLY TO YOU.  THIS WARRANTY GIVES YOU SPECIFIC LEGAL RIGHTS AND YOU MAY 
+ALSO HAVE OTHER LEGAL RIGHTS WHICH VARY FROM STATE TO STATE.
+ 
+
+7.  CUSTOMER REMEDIES
+ 
+YOUR SOLE AND EXCLUSIVE REMEDY AND ML'S SOLE LIABILITY HEREUNDER 
+SHALL BE, UPON YOUR RETURN OF THE SOFTWARE TO ML WITH THE RECEIPT 
+WITHIN THE WARRANTY PERIOD, AT ML'S OPTION, REPLACEMENT OR REPAIR 
+OF SOFTWARE THAT DOES NOT MEET ML'S LIMITED WARRANTY OR A REFUND OF 
+THE SOFTWARE'S PURCHASE PRICE.  ML'S SUPPLIERS SHALL NOT BE HELD TO 
+ANY LIABILITY FOR ANY DAMAGES SUFFERED OR INCURRED BY YOU.  IN NO EVENT 
+SHALL ML OR ITS SUPPLIERS BE LIABLE TO YOU OR ANY OTHER PERSON FOR ANY 
+INCIDENTAL, COLLATERAL, SPECIAL, OR CONSEQUENTIAL DAMAGES, INCLUDING 
+WITHOUT LIMITATION, DAMAGES FOR LOSS OF PROFITS, LOSS OF CUSTOMERS, 
+LOSS OF GOOD WILL, WORK STOPPAGE, DATA LOSS, COMPUTER FAILURE OR 
+MALFUNCTION, CLAIMS BY ANY PARTY OTHER THAN YOU, OR ANY OR ALL OTHER 
+SIMILAR DAMAGES OR LOSS EVEN IF ML, ITS SUPPLIERS, OR ITS AGENT HAS 
+BEEN ADVISED OF THE POSSIBILITY OF SUCH DAMAGES.  SOME STATES DO NOT 
+ALLOW THE EXCLUSION OR LIMITATION OF INCIDENTAL OR CONSEQUENTIAL 
+DAMAGES, SO THE ABOVE LIMITATIONS OR EXCLUSION MAY NOT APPLY TO YOU.
+
+ML shall have no liability or responsibility for Software altered, 
+modified, or converted by you or a third party, damages resulting 
+from accident, abuse, or misapplication, or for problems due to the 
+malfunction of your equipment or software not supplied by ML.
+ 
+
+8.  TERMINATION
+ 
+This License is in effect until terminated and terminates without 
+notice from ML if you fail to comply with any of its provisions.  
+Upon termination you shall either destroy the Software and all copies 
+or portions thereof or return the Software and all copies as portions 
+thereof to ML.
+ 
+
+9.  GENERAL
+ 
+This License is governed by the laws of the State of Florida.  This 
+License represents the entire statement of the understandings and 
+agreements among the parties.
+ 
+
+10. U.S. GOVERNMENT RESTRICTED RIGHTS
+ 
+This Software and documentation is provided with RESTRICTED RIGHTS.  
+Use, duplication, or disclosure by the Government is subject to 
+restrictions as set forth in subparagraph (c)(1)(ii) of the Rights 
+in Technical Data and Computer FAR 52.227-14 (Alternate III) or the 
+Software clause at DFAR 252.227-7013.  Contractor/manufacturer is 
+METRO LINK INCORPORATED, 4711 North Powerline Road, Fort Lauderdale, 
+FL  33309.
+
+Metro Link Motif Complete!, Metro Link Motif, Metro-X, Metro
+Extreme 3D, Metro OpenGL, Metro-Xv, and Metro-XIE are trademarks
+of Metro Link Incorporated.
+
+XRT, XRT/graph, XRT/field, XRT/gear, XRT/3d, and XRT/table are
+trademarks of KL Group Incorporated.
--- motif-devel-2.1.10.orig/debian/rules
+++ motif-devel-2.1.10/debian/rules
@@ -0,0 +1,87 @@
+#!/usr/bin/make -f
+# debian/rules that uses debhelper and alien
+# GNU copyright 1997 by Joey Hess.
+
+SHELL = /bin/sh
+
+package := $(shell basename `pwd` | sed 's/-[0-9].*//')
+version := $(shell basename `pwd` | tr -cd '[0-9.]')
+major := $(word 1,$(subst ., ,$(version)))
+minor := $(word 2,$(subst ., ,$(version)))
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+# This is the debhelper compatability version to use.
+export DH_COMPAT=1
+
+motif-version:
+	sed -e 's/#PACKAGE#/$(package)/' -e 's/#VERS#/$(version)/' debian/changelog-template >debian/changelog
+
+build: motif-version
+	dh_testdir
+
+clean: motif-version
+	dh_testdir
+	dh_testroot
+	dh_clean
+
+# Build architecture-independent files here.
+binary-indep: build
+# We have nothing to do by default.
+
+# Build architecture-dependent files here.
+binary-arch: build
+#	dh_testversion
+	dh_testdir
+	dh_testroot
+	dh_clean -k
+	dh_installdirs
+	tar -cf - --exclude=debian --exclude='./usr/doc' . | tar -xpf - -C debian/tmp
+#
+# If you need to move files around in debian/tmp or do some
+# binary patching ... Insert it here
+#
+	-ln -fs libMrm.so.$(major).$(minor) debian/tmp/usr/X11R6/lib/libMrm.so
+	-ln -fs libUil.so.$(major).$(minor) debian/tmp/usr/X11R6/lib/libUil.so
+	-ln -fs libXm.so.$(major).$(minor) debian/tmp/usr/X11R6/lib/libXm.so
+# Move /usr/X11R6/lib/X11/app-defaults to /etc/X11/app-defaults.
+	@if [ -d debian/tmp/usr/X11R6/lib/X11/app-defaults ]; then \
+	  echo "Moving /usr/X11R6/lib/X11/app-defaults to /etc/X11/app-defaults"; \
+	  [ -d debian/tmp/etc/X11 ] || mkdir -p debian/tmp/etc/X11; \
+	  mv debian/tmp/usr/X11R6/lib/X11/app-defaults debian/tmp/etc/X11; \
+	fi
+# Install mxmkmf.
+	@if [ ! -e debian/tmp/usr/X11R6/bin/mxmkmf ]; then \
+	  echo "Installing mxmkmf"; \
+	  install -m 0755 debian/mxmkmf debian/tmp/usr/X11R6/bin/mxmkmf; \
+	fi
+# Copy documentation.
+	@if [ -d usr/doc ]; then \
+	  echo "Copying documentation"; \
+	  mkdir -p debian/tmp/usr/share/doc/$(package); \
+	  cp -a usr/doc/*/* debian/tmp/usr/share/doc/$(package); \
+	fi
+	dh_installdocs
+	dh_installexamples
+	dh_installmenu
+#	dh_installinit
+#	dh_installcron
+#	dh_installmanpages
+#	dh_undocumented
+	dh_installchangelogs
+	dh_strip
+	dh_compress
+	dh_fixperms
+	dh_installdeb
+	-dh_shlibdeps
+	dh_gencontrol
+#	dh_makeshlibs
+	dh_md5sums
+	dh_builddeb
+
+source diff:                                                                  
+	@echo >&2 'source and diff are obsolete - use dpkg-source -b'; false
+
+binary: binary-indep binary-arch
+.PHONY: build clean binary-indep binary-arch binary motif-version
--- motif-devel-2.1.10.orig/debian/mxmkmf
+++ motif-devel-2.1.10/debian/mxmkmf
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+# $XFree86$
+#
+# make a Makefile from an Imakefile from inside or outside the sources
+# 
+# $XConsortium: xmkmf.cpp /main/22 1996/09/28 16:17:05 rws $
+
+usage="usage:  $0 [-a] [top_of_sources_pathname [current_directory]]"
+
+configdirspec="-I/usr/X11R6/lib/X11/config"
+topdir=
+curdir=.
+do_all=
+imake_defines="-DProjectRulesFile=<Motif.rules.metro> -DProjectTmplFile=<Motif.tmpl.metro>"
+
+while [ $# -gt 0 ]
+do
+    case "$1" in
+    -D*)
+       	imake_defines="$imake_defines $1"
+	shift
+	;;
+    -a)
+	do_all="yes"
+	shift
+	;;
+    *)
+    	break
+	;;
+    esac
+done
+
+case $# in 
+    0) ;;
+    1) topdir=$1 ;;
+    2) topdir=$1  curdir=$2 ;;
+    *) echo "$usage" 1>&2; exit 1 ;;
+esac
+
+case "$topdir" in
+    -*) echo "$usage" 1>&2; exit 1 ;;
+esac
+
+if [ -f Makefile ]; then 
+    echo mv -f Makefile Makefile.bak
+    mv -f Makefile Makefile.bak
+fi
+
+if [ "$topdir" = "" ]; then
+    args="-DUseInstalled "$configdirspec
+else
+    args="-I$topdir/config/cf -DTOPDIR=$topdir -DCURDIR=$curdir"
+fi
+
+echo imake $imake_defines $args
+case "$do_all" in
+yes)
+    imake $imake_defines $args && 
+    echo "make Makefiles" &&
+    make Makefiles &&
+    echo "make includes" &&
+    make includes &&
+    echo "make depend" &&
+    make depend
+    ;;
+*)
+    imake $imake_defines $args
+    ;;
+esac
--- motif-devel-2.1.10.orig/debian/dirs
+++ motif-devel-2.1.10/debian/dirs
@@ -0,0 +1,2 @@
+usr/X11R6/bin
+usr/X11R6/lib
--- motif-devel-2.1.10.orig/debian/README.Debian
+++ motif-devel-2.1.10/debian/README.Debian
@@ -0,0 +1,6 @@
+Motif for Debian
+----------------
+
+Please use mxmkmf instead of xmkmf to build Motif programs.
+
+ -- Andreas Voegele <andreas.voegele@gmx.de>  Tue, 16 Nov 1999 19:31:18 +0100
