--- xf86config-glibc-1.0.0.i386.orig/debian/changelog
+++ xf86config-glibc-1.0.0.i386/debian/changelog
@@ -0,0 +1,10 @@
+xf86config-glibc (1.0.0.i386-1) unstable; urgency=low
+
+  * Added preinst and postrm for xf86config and Card files diversion
+  * Converted from Slackware .tgz binary format to debian
+
+ -- Agustín Martín Domingo <agmartin@aq.upm.es>  Thu, 15 Apr 1999 15:47:14 +0200
+
+Local variables:
+mode: debian-changelog
+End:
--- xf86config-glibc-1.0.0.i386.orig/debian/control
+++ xf86config-glibc-1.0.0.i386/debian/control
@@ -0,0 +1,10 @@
+Source: xf86config-glibc
+Section: unknown
+Priority: extra
+Maintainer: Agustín Martín Domingo <agmartin@aq.upm.es>
+
+Package: xf86config-i740g
+Architecture: any
+Depends: ${shlibs:Depends}, xbf-i740-glibc
+Description: xf86config for Intel i740 based graphics cards
+ xf86 configuration tool for Intel i740 chipset based graphic cards
--- xf86config-glibc-1.0.0.i386.orig/debian/copyright
+++ xf86config-glibc-1.0.0.i386/debian/copyright
@@ -0,0 +1,4 @@
+This package was debianized by the alien program by converting
+a binary Slackware tgz Package on Thu, 15 Apr 1999 15:47:14 +0200.
+
+Copyright: Unknown
--- xf86config-glibc-1.0.0.i386.orig/debian/rules
+++ xf86config-glibc-1.0.0.i386/debian/rules
@@ -0,0 +1,57 @@
+#!/usr/bin/make -f
+# debian/rules that uses debhelper and alien
+# GNU copyright 1997 by Joey Hess.
+
+# Uncomment this to turn on verbose mode.
+#export DH_VERBOSE=1
+
+build:
+	dh_testdir
+	# Nothing to do.
+
+clean:
+	dh_testdir
+	dh_testroot
+	dh_clean
+
+# Build architecture-independent files here.
+binary-indep: build
+# We have nothing to do by default.
+
+# Build architecture-dependent files here.
+binary-arch: build
+#	dh_testversion
+	dh_testdir
+	dh_testroot
+	dh_clean -k
+	dh_installdirs
+	install -d debian/tmp
+	cp -a `ls |grep -v debian` debian/tmp
+#
+# If you need to move files around in debian/tmp or do some
+# binary patching ... Insert it here
+#
+	dh_installdocs
+	dh_installexamples
+	dh_installmenu
+#	dh_installinit
+	dh_installcron
+	dh_installmanpages
+#	dh_undocumented
+	dh_installchangelogs
+#	dh_strip
+	dh_compress
+	dh_fixperms
+	dh_suidregister
+	dh_installdeb
+	dh_shlibdeps
+	dh_gencontrol
+	dh_makeshlibs
+	dh_md5sums
+	dh_builddeb
+
+source diff:                                                                  
+	@echo >&2 'source and diff are obsolete - use dpkg-source -b'; false
+
+binary: binary-indep binary-arch
+.PHONY: build clean binary-indep binary-arch binary
--- xf86config-glibc-1.0.0.i386.orig/debian/preinst
+++ xf86config-glibc-1.0.0.i386/debian/preinst
@@ -0,0 +1,8 @@
+#!/bin/sh -e 
+
+if [ install = "$1" ]; then
+   dpkg-divert --package  xf86config-i740g --add --rename \
+	--divert /usr/X11R6/bin/xf86config.dpkg-old /usr/X11R6/bin/xf86config
+   dpkg-divert --package  xf86config-i740g --add --rename \
+	--divert /usr/X11R6/lib/X11/Cards.dpkg-old /usr/X11R6/lib/X11/Cards
+fi
--- xf86config-glibc-1.0.0.i386.orig/debian/postrm
+++ xf86config-glibc-1.0.0.i386/debian/postrm
@@ -0,0 +1,8 @@
+#!/bin/sh -e
+
+if [ remove = "$1" ]; then
+  dpkg-divert --package xf86config-i740g --remove --rename \
+	--divert /usr/X11R6/bin/xf86config.dpkg-old /usr/X11R6/bin/xf86config
+  dpkg-divert --package xf86config-i740g --remove --rename \
+	--divert /usr/X11R6/lib/X11/Cards.dpkg-old /usr/X11R6/lib/X11/Cards 
+fi
